_posts/2020-02-28-newpipe-0.18.6-released.md

```diff
@@ -0,0 +1,49 @@
+date: 2020-02-28 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Have you fully read the last blog post and remember the [first paragraph in the Whats New]({% link _posts/2020-02-26-newpipe-0.18.5-released.md %}#what-changes) section? Do you not and you can't care less? Well, I don't care either, I have a story for both of you...
+<!-- more -->
+
+## Another release???
+
+Yes, another release. The developers told me they are deeply sorry. And now it is on me to try to explain why you have to update (and I have to write a blogpost...) again.
+
+The build two days ago was a preparation for March, where YouTube shuts down the way NewPipe used to access their website. The release happened as quickly as possible in order to get the update out in time to F-Droid users, since NewPipe basically faces a wait-time of a week for each update.
+
+Now, as you can imagine, basically every part of the app relies in one way or another on a properly parsed YouTube site. After pushing out the update and ensuring in advance that basic funcionalities do work with the new way of accessing YouTube, users and developers where quick to realize that not everything works as well as it used to. So the NewPipe team decided to push out a hotfix for the hotifx, adressing these issues.
+
+## What changes
+
+A lot. All of the entries in the list fix issues not happening in 0.18.4. We have two general improvements though. [@Stypox](https://github.com/Stypox) improved the playback. You are already able to increase the tempo and pitch, and can unlink those settings. So far, you would need to unlink them in every video. Now, NewPipe remembers what you selected, so you can watch every video as linked or unlinked as you like.
+If you happen to have one of those fancy fullscreen smartphones with a notch, you can thank [@moneytoo](https://github.com/moneytoo), he made NewPipe support them in fullscreen properly. Now, to the list of fixes, you will love it:
+
+- Proper rounding of number of subscribers
+- Trending page not containing everything, not only three elements. To be honest, who is actually watching those? Anyway, if you are, you can without any restrictions now. Lucky you.
+- Correctly parse special links, not to special anymore
+- Pictures of certain channels werent displayed
+- YouTube Music playlists work again (yay music)
+- Some channels don't have videos tabs, which doesn't break NewPipe anymore (take that weird channels)
+- Video titles with <code>@</code> work again
+- <code>#</code> in descriptions are supported again
+- Channel descriptions work again
+- 0 subscribers on channels - 0 broken NewPipes. At least now
+- Fix channels with multiple views on Videos tab
+- Thumbnail URLs starting with // work again
+- All search queries which should return something will return something
+- Correctly display Network error when a network error happened instead of a general crash error (so much error)
+
```

Comment:
No mention of reduced data usage due to using pbj=1? See https://github.com/TeamNewPipe/NewPipeExtractor/pull/262#issuecomment-592936948 for my explanation.